src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -68,6 +68,9 @@ class AdapterSpec:
     # set of training instances.  Used to compute error bars.
     num_train_trials: int = 1
 
+    # Number of trials, where we query the model with the exact same requests
+    num_random_trials: int = 1
```

Comment:
Isn't this equivalent to `num_outputs`?

Reply:
`num_outputs` is the number of outputs for a single request. `num_random_trials` is the number of calls for the same requests.